examples/thoughts/ext/Layout.css

```diff
@@ -0,0 +1,12 @@
+.base-page {
```

Comment:
You took `main-page` and `main-container` names and turned them into `base-page` and `base-container`.
I wonder if this is a good opportunity to find them even better names?
Maybe `layout-container`/`layout-root` and `layout-content`?

Reply:
I like `layout-root` and `layout-content`. Changed these now.